client/src/components/Button.js

```diff
@@ -0,0 +1,16 @@
+import styled from "styled-components/macro";
+
+const Button = styled.button`
+  height: 3rem;
+  width: 150px;
+  border-radius: 6px;
+  background: var(--secondary-success);
+  border: none;
+  cursor: pointer;
+  color: var(--primary-third);
+  font-weight: bold;
+  margin: 1.5em 0 3em 0;
+  filter: drop-shadow(2px 2px 2px var(--neutral-mid-N60));
```

Comment:
I'd suggest to name that global var to something more clear, like neutral-blue or whatever color it might be ;)

Reply:
I named the colors like this in case I want to change them later. Therefore I decided a more generic variable would be a better option.